flaml/nlp/autotransformers.py

```diff
@@ -805,14 +801,15 @@ def predict(self,
             try:
                 self.test_dataset.remove_columns_("label")
             except ValueError:
+                print("No such label in column")
```

Comment:
does a user understand what 'such label' means?

Reply:
This exception happens when removing the label from the test data, but sometimes the test data doesn't contain the label. I decided user doesn't need to know this.

Reply:
Even for developers, this may be confusing.